Extension Project_Final.Rmd

```diff
@@ -0,0 +1,876 @@
+---
+title: "Extension of Villarreal Paper"
+author: "Reisman and Lajous"
+date: "April 23, 2015"
+output: html_document
+---
+
```

Comment:
I read through the diffs (https://github.com/soc504-s2015-princeton/replication/pull/5/files) and your changes all look good. I think the code is a bit clearer now. :+1: